Tests/conf.json

```diff
@@ -4866,10 +4874,8 @@
         "EWS Extension: Powershell Online V2 Test": "Issue 39008",
         "O365 - EWS - Extension - Test": "Issue 39008",
         "Majestic Million Test Playbook": "Issue 30931",
-        "iDefense_v2_Test": "Issue 40126",
         "EWS Mail Sender Test": "Issue 27944",
         "McAfee ESM v2 - Test v10.3.0": "Issue 35616",
-        "Feed iDefense Test": "Issue 34035",
```

Comment:
has [this issue](https://github.com/demisto/etc/issues/34035) been resolved, allowing for un-ignoring this one? @kgal-pan